packages/nuxt/cypress.json

```diff
@@ -2,5 +2,7 @@
   "baseUrl": "http://localhost:3000",
   "viewportWidth": 1400,
   "viewportHeight": 960,
-  "video": false
+  "video": false,
+  "pageLoadTimeout": 180000,
+  "defaultCommandTimeout": 10000
```

Comment:
@RafalDobrowolski I'm wondering if it would be better to add the timeouts to SfHero test where the problem may be because now we postpone all commands and the test will take significantly longer, wdyt?

Reply:
OK, let's check if this works smoothly without big delays :+1: